swagger/public_v1/api_spec.json

```diff
@@ -1926,7 +1926,7 @@
             "name": "year",
             "in": "path",
             "required": true,
-            "description": "The starting year of the recruitment cycle.",
+            "description": "The starting year of the recruitment cycle. Also accepts \"current\" for the current recruitment cycle.",
             "example": "2020",
```

Comment:
minor 🤏  `2023` has been bumped to `2024`, but `2020` has been left in place in a couple of examples. If we're touching the docs already, I wonder if it's good to have a clean slate and move everything to `2024`!

Reply:
Changed in bf576799da257a2ceef7d279c1b994aad1afe1e8